modules/visualization/dcv-k8s-deployment/README.md

```diff
@@ -0,0 +1,58 @@
+# DCV-EKS Module
+
+## Description
+
+## Deployment
+
+### Prerequisites
+
+- AWS SecretsManager secret `dcv-credentials` with UserName and Password already established (as described in the dcv-image module)
+
+#### Using AWS SecretsManager
+
+The secret `dcv-credentials` needs to exist with key `UserName` and `Password` keys.
+
+## Inputs/Outputs
+
+### Input Paramenters
+
+#### Required
+
+- `dcv-image-uri` - DCV container image uri
+- `eks-cluster-admin-role-arn` - the role which creates the eks cluster
+- `eks-handler-role-arn` - The IAM role of EKS Cluster handler for running kubectl commands
+- `eks-cluster-name` - the name of the EKS cluster
+- `eks-oidc-arn` - full ARN of the OIDC provider
+- `eks-cluster-open-id-connect-issuer` - OIDC provider URI
+- `eks-cluster-security-group-id` - id of security group which is attached to all nodes in eks
+- `eks-node-role-arn` - arn of the role which is attached to all nodes in eks
+
+#### Optional
+
+- `dcv-namespace`: the namespace to store all DCV related resources. Defaults to `dcv`.
+
+### Module Metadata Outputs
+
+- `DcvEksRoleArn`: arn of the role
+- `DcvNamespace`: the namespace to create all DCV resources
+
+- `DcvDisplayParameterName`: SSM parameter name for display number
+- `DcvSocketMountPathParameterName`: SSM parameter name for shared directory path in worker node
+
+#### Output Example
+
+```json
+{
+    "DcvEksRoleArn": "arn:aws:iam::XXXXXXXX:role/DcvEksRole",
+    "DcvNamespace": "dcv",
+    "DcvDisplayParameterName": "/addf/mlops/dcv-eks-dcv-eks/dcv-display",
+    "DcvSocketMountPathParameterName": "/addf/mlops/dcv-eks-dcv-eks/dcv-socket-mount-path"
+}
+
```

Comment:
Nit: this is missing closing backticks so all below is highlighted by syntax checker